src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
```

Comment:
My problem is that it implies a shape to the data which doesn't need to be there.